waiter/src/waiter/core.clj

```diff
@@ -1126,11 +1126,11 @@
                                      [:routines prepend-waiter-url router-metrics-helpers service-id->service-description-fn]
                                      [:state entitlement-manager]
                                      wrap-secure-request-fn]
-                              (let [state-chan (get-in router-state-maintainer [:maintainer-chans :state-chan])
+                              (let [query-state-fn (get-in router-state-maintainer [:maintainer-chans :query-state-fn])
```

Comment:
Feels awkward to have `:query-state-fn` inside a map called `maintainer-chans`.  It's not a `chan` anymore.

Reply:
Renamed `maintainer-chans` to `maintainer`.